js/webpack_get_defaults.js

```diff
@@ -132,7 +139,7 @@ module.exports = function (basePath) {
       loaders: [
         {
           test: /\.js/,
-          loader: combineLoaders(jsLoaders),
+          loader: 'babel?{"presets":["es2015","stage-3"],"plugins":["transform-strict-mode",["transform-runtime",{"polyfill":true,"regenerator":false}]]}',
```

Comment:
Yes I did not intend to let it as it is, that's messy :). I dove into url stringification and couldn't figure out why it didn't work so I'd hope you guys would have an idea. I'll dig more !